ci/infra/groups.yml

```diff
@@ -0,0 +1,17 @@
+groups:
+  - name: AL2 AMIs
```

Comment:
personal opinion: it would be tidier if all the PR jobs were in their own group

Reply:
Done.